tests/flowbits-prefilter-13-tx-onedir-toggle/test.yaml

```diff
@@ -0,0 +1,34 @@
+requires:
+  min-version: 8
+
+pcap: ../flowbit-oring/input.pcap
+
+args:
+  - -k none
+  - --simulate-ips
+
+checks:
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      pcap_cnt: 6
+      alert.signature_id: 11
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      pcap_cnt: 6
+      alert.signature_id: 22
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      pcap_cnt: 6
+      alert.signature_id: 25
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      pcap_cnt: 6
+      alert.signature_id: 33
```

Comment:
Should not we have engine-analysis tests as well ?

Reply:
what would you want to test for specifically?

Can do it as a follow up